modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,108 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.2.0
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.2 that need your attention. You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Please note that this minor release contains database migrations which have an impact on the upgrade duration:
+
+- The table `oc_share` receives a new column (impact depends on the number of shares)
+- The `oc_authtoken` table's login name column size has been increased (impact depends on the number of recently logged in users plus number of app passwords)
+
+=== PHP 5.6/7.0 Deprecation
+
+Following the https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation[PHP 5.6/7.0 deprecation note with the release of ownCloud Server 10.0.8], ownCloud Server 10.2 *does not support PHP 5.6* anymore. If you're still running on 5.6 it is necessary to upgrade to PHP 7 before upgrading to ownCloud Server 10.2. See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud documentation]. Also some new app versions do not support older PHP versions anymore. These will not upgrade if your PHP version is not compatible.
+
+To provide additional time for upgrades, version 10.2 still supports PHP 7.0 as some major Linux distributions continue to deliver support. To ensure secure and performant operations, support for PHP 7.0 will also be discontinued within the next minor versions of ownCloud 10. Please plan an upgrade soon.
+
+Apart from this, ownCloud Server is also being prepared for PHP 7.3 support which will become available with the next minor versions.
```

Comment:
Do you intend to be a bit "flexible"/non-committal by saying "with the next minor versions"?
Maybe it reads better with either:
"with the next minor version"
or
"with an upcoming minor version release"